schemas/com.snowplowanalytics.iglu/igluctl_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,201 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for Igluctl's configuration",
+	"self": {
+		"vendor": "com.snowplowanalytics.iglu",
+		"name": "igluctl_config",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"env": {
```

Comment:
I'm still hazy on this field. What would be the semantic difference? Does Igluctl behave differently on various envs?

Reply:
We expect users to have multiple configs and `env` field would be used to identify a config between multiple ones. Igluctl's behavior doesn't change per `env`. 

Should we rely on only file name to identify which config file will be used for which environment (test, mini, prod)?

Reply:
Yes, I'd expect users rely on filename only. We could have something like `description` to say that "This config was used for initial com.acme testing on Mini 0.5.0", but just `test`/`prod` surely not enough.

So we can leave human readable `name` or `description`, but don't think `env` fits the purpose.